aries-site/src/pages/templates/cards.mdx

```diff
@@ -1,3 +1,113 @@
-import { ComingSoon } from '../../components';
 
-<ComingSoon />
+import { Example } from '../../layouts';
+import { 
+  CardsExample, 
+  CardsFooterExample,
+  CardsIconIdentifierExample, 
+  CardsIconIdentifierVertExample,
+  CardsImageIdentifierExample, 
+  CardsPreviewGrid,
+  CardsSimpleExample 
+} from '../../examples/templates/cards';
+
+Cards formats are versatile, serving a wide variety of use cases. Explore the 
+following examples to gain a feel for how these formats may be applied.
+
+<CardsPreviewGrid />
+
+## Presentation ##
```

Comment:
Would we want to continue with our pattern of using the term `Guidance` for this section? soft opinion here but thinking of people who might be used to that pattern